### ISSUE TITLE
Set variable based on json property of object in array

### ISSUE THREAD

**Issue by dw72** [rangav/thunder-client-support#846]
I have JSON data in response like this:

```
{
  "data": {
    "items": [
      {
        "id": "001",
        "name": "one"
      },
      {
        "id": "003",
        "name": "three"
      },
      {
        "id": "004",
        "name": "four"
      }
    ],
    "total_affected_items": 3,
    "total_failed_items": 0,
    "failed_items": []
}
```

I want to set variable which contains array of all items.id, tried different queries ex. `json.data.items[].id` and all of them not working. How I should write this to have in variable list od id's `001,003,004`?

**Comment by rangav** [comment]
@dw72 currently not possible will add to roadmap.

**Comment by mauriliofilho** [comment]
@dw72 you have try this json.data.items[?(@.id)]

**Comment by rangav** [comment]
This feature is now implemented and published to marketplace, please update to `v2.0.0`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.0.0

Please let me know your feedback.

You needed to use `map` filter now.
<img width="742" alt="Screenshot 2022-11-17 at 11 50 03" src="https://user-images.githubusercontent.com/8637550/202438952-478309c3-0c4c-4982-bcda-b244c27c33b8.png">

**Comment by rangav** [comment]
We added new array filter using `[*]` in `v2.0.1`

In your use-case you can use below
`json.data.items[*].id | join`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.0.1

Please let me know your feedback.
